liana-gui/src/app/view/settings.rs

```diff
@@ -583,6 +584,16 @@ pub fn electrum_edit<'a>(
             .push(separation().width(Length::Fill));
     }
 
+    let checkbox = checkbox(
+        "Do not validate SSL Domain(check this only if you want to use a self-signed certificate)",
+        !validate_domain,
+    )
+    .on_toggle(|b| SettingsEditMessage::ValidateDomainEdited(!b));
+    let checkbox = if addr.valid && addr.value.contains("ssl://") {
```

Comment:
nit: The function `is_electrum_address_valid(value: &str)` uses `value.starts_with("ssl://")` so you could use the same condition here (or even add it to a standalone `is_ssl()` function and then use it within `is_electrum_address_valid`).

Also, to avoid repeating this checkbox logic in the installer, you could add it to src/node/electrum.rs.